### PR TITLE
InitialOrigin will be used to pair the setCookie with the eventual non-redirect host

### PR DESCRIPTION
**Pull request by scottjehl**

### CODE REVIEW

**Comment by scottjehl** [comment]
showing in this case, an origin without www and the host and cookie picking up the eventual redirected www to use
image

File: www/runtest.php [hunk 1]
```diff
@@ -839,6 +839,7 @@ function buildSelfHost($hosts){
                   } else { 
                     // the first non-redirect host is passed in from experiments
                     $hostToUse = isset( $req_initialHostNonRedirect ) ? $req_initialHostNonRedirect : '%HOST%';
+                    $originToUse = isset( $req_initialOriginNonRedirect ) ? $req_initialOriginNonRedirect : '%ORIGIN%';
```
Comment (jefflembeck) [inline www/runtest.php:842]:
```suggestion
                    $originToUse = $req_initialOriginNonRedirect ?? '%ORIGIN%';
```

Comment (jefflembeck) [reply www/runtest.php:842]: you don't have to do this, but it's a nice shorthand

**Review by jefflembeck — APPROVED** [review]
Logic seems sound

**Pull request merged**